app/controllers/newflow/instructor_signup_controller.rb

```diff
@@ -1,12 +1,12 @@
 module Newflow
-  class EducatorSignupController < SignupController
+  class InstructorSignupController < SignupController
 
-    include EducatorSignupHelper
+    include InstructorSignupHelper
 
     skip_forgery_protection(only: :sheerid_webhook)
 
     before_action(:prevent_caching, only: :sheerid_webhook)
-    before_action(:exit_newflow_signup_if_logged_in, only: :educator_signup_form)
+    before_action(:exit_newflow_signup_if_logged_in, only: :instructor_signup_form)
```

Comment:
First - thanks for tackling this!
I had originally thought to change `educator` to `instructor` to make sure I had got everything.. but since we call this `instructor` in Salesforce, I think it makes the most sense for consistency too. Just a little background on the change you saw in my PR.